src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      data-qa="big"
+      action="#"
+      method="post"
+    >
+      <label class="label1">
+        <img
+          class="image1"
+          src="./images/Search.svg"
+          alt="Search"
+        >
+
+        <input
+          data-qa="keypress"
+          class="text1"
+          type="text"
+          name="img"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form
+      data-qa="small"
+      action="#"
+      method="post"
+    >
+      <label class="label2">
+        <img
+           class="image2"
+           src="./images/Search.svg"
+           alt="Search"
+         >
+        <input
+           class="text2"
+           type="text"
+           name="img"
+           placeholder="Try &quot;Los Angeles&quot;"
+         >
```

Comment:
- opening and closing tag should be at the same level
- you need 2 indents (you had 3) after moving the attribute to the new line
```suggestion
        <img
          class="image2"
          src="./images/Search.svg"
          alt="Search"
        >
        <input
          class="text2"
          type="text"
          name="img"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```

Reply:
i will fix it :)